### ISSUE TITLE
[Feature Request] Add better instructions for leaving, or following road forks, on autobahns

### ISSUE THREAD

**Issue by keksimusmaximus** [mapsme/omim#11257]
Hello,

currently it's sometimes hard to follow the navigation instructions when traveling on autobahns. Especially when it comes to leaving them or following street forks to switch autobahns, I often got confused and took the wrong route.

It would be useful to add instructions like "on the next road fork, stay [left,right] to drive towards x". Also a graphical overviews would be very nice, like others do it. (But I think it gets difficult bc maps.me is relying on openstreetmap data here (which is a nice thing!)).

Examples here: 
https://i.postimg.cc/vm9K8gk0/maps-me.png
https://i.postimg.cc/7hPtNB6R/maps-me2.jpg
https://i.postimg.cc/76nQ0vx7/maps-me3.jpg

**Comment by bykoianko** [comment]
Hello, @keksimusmaximus ,

I agree with you. I'll be better to have more variants of turn instructions in Maps.Me. Probably we add some of them in the future. Concerning leaving autobahns, there's now a special turn instruction: 
![image](https://user-images.githubusercontent.com/1768114/61618679-ff8b9680-ac75-11e9-930a-c01484213a9c.png)

Regards,
Vladimir

**Comment by thany** [comment]
I personally like the Google Maps way most:
It just shows all the lanes and highlights the one you should be in.

I must say the examples you gave, @keksimusmaximus, apply to motorways. Lane highlighting is useful for non-motorways as well.

It should also probably combine turns/exits that should be taken in quick succession. Some part of the Japanese expressways inside the cities may have three or four possible exists in *very* quick succession, which is very confusing when you're not familiar with Japanese roads. Just an example.

**Comment by bitboy85** [comment]
Some Sort of Reality View (like in the shown pictures) would be amazing!
